apps/vs-code-designer/src/app/utils/packageManagers/hasBrew.ts

```diff
@@ -0,0 +1,31 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { localize } from '../../../localize';
+import { executeCommand } from '../funcCoreTools/cpUtils';
+import { tryGetInstalledBrewPackageName } from './getBrewPackageName';
+
+/**
+ * Checks if the system has brew installed.
+ * @param {boolean} isPackageInstalled - Is package Installed.
+ * * @param {string} brewPackageName - Brew Package Name.
+ * @returns {Promise<boolean>} Returns true if the system has brew installed, otherwise returns false.
+ */
+export async function hasBrew(isDotNetSDKInstalled: boolean, brewPackageName: string): Promise<boolean> {
```

Comment:
NIT: I kind of feel that we can use the `hasBrew` function that already exits, because if we sent false the param will always execute the brew command. Or maybe update the other one to not be 100% stick to function core tools. Lets discuss this as maybe I don't have all the context behind this